client/src/ingredients/browser-compatibility-table/browser-support-notes.js

```diff
@@ -9,6 +9,19 @@ function shouldDisplayBlock(blockDisplayed, versionAdded) {
   return true;
 }
 
+function flagInfoForBrowser(browser) {
+  switch(browser) {
+    case 'chrome':
+    case 'chrome_android':
+      return ' To change preferences in Chrome, visit chrome://flags.';
+    case 'firefox':
+    case 'firefox_android':
+      return ' To change preferences in Firefox, visit about:config.';
```

Comment:
These are eventually going to have to go into a `gettext` functionality so they can be localized. So you'll need to remove that extra prefix whitespace.